apps/ewallet_db/lib/ewallet_db/auth_token.ex

```diff
@@ -218,6 +233,18 @@ defmodule EWalletDB.AuthToken do
     :ok
   end
 
+  def set_required_2fa_for_user(user, required_2fa) do
```

Comment:
I'm a bit scared of manipulating `AuthToken` and potentially having `AuthToken` that may and may not have permissions.

Shall we just delete all auth_tokens of the user when enabling 2FA?

What do you think? @ripzery @T-Dnzt @sirn

Reply:
Yeah, I like the idea of logging out the users from any current session by deleting the existing tokens.